app/controllers/admin/collections_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Admin
+  # Generates collection reports
+  class CollectionsController < ApplicationController
```

Comment:
Can this be CollectionReportsController?

Reply:
It can, but this matches the naming conventions of `UsersController`.

Reply:
For UsersController, the resource being controlled was a User, in this case the resource being controlled is the CollectionReport (a report about collections).

Reply:
Changed.